Makefile

```diff
@@ -23,10 +31,10 @@ test: development install-hooks
 	./venv/bin/coverage html
 	./venv/bin/pre-commit run --all-files
 	./venv/bin/check-requirements
+	npm run eslint
 	npm test
-	node_modules/.bin/eslint .
```

Comment:
Apparently, this wasn't checking JSX files. Found that out when running `./node_modules/.bin/eslint .` wasn't surfacing errors the Webpack `eslint-loader` was throwing. Had to use the eslint `--ext` option to explicitly tell eslint to check JSX files.